Makefile

```diff
@@ -5,6 +5,7 @@ BASE_IMAGE ?= gitops-service
 USERNAME ?= redhat-appstudio
 IMG ?= quay.io/${USERNAME}/${BASE_IMAGE}:${TAG}
 APPLICATION_API_COMMIT ?= 394e8c127f031990838397fcb2d1e10110b2e7ae
+DB_PASS ?= gitops
```

Comment:
Hey Jayendra, is this needed? If possible, I'd prefer not to hardcode DB_PASS to any particular value. (We are using a hardcoded DB_PASS for the local case, but in that case the postgresql server shouldn't be public facing)

Reply:
I removed it, since we are not passing DB_PASS as an argument anymore.